.github/workflows/multi-node-test-workflow.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - name: Run integration tests with multi node config
         run: |
           chown -R 1000:1000 `pwd`
-          su `id -un 1000` -c "./gradlew integTest -PnumNodes=5"
+          su `id -un 1000` -c "./gradlew integTest -PnumNodes=3"
```

Comment:
I changed this to 3 because GH was showing an OOM error when running the tests with 5.